### PR TITLE
[13.0] [MIG] stock_move_value_report: Migration to 13.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT31776

please @chienandalu @carlosdauden, review this

### CODE REVIEW

File: stock_move_value_report/report/stock_move_line_value_report.xml [hunk 1]
```diff
@@ -84,8 +92,23 @@
...
                             <td class="text-right">
+                                <t t-set="total" t-value="0" />
+                                <t t-foreach="move_lines" t-as="move_line">
+                                    <t
+                                        t-set="valuation_layers"
+                                        t-value="move_line.move_id.stock_valuation_layer_ids"
+                                    />
+                                    <t
+                                        t-set="cost"
+                                        t-value="sum(valuation_layers.mapped(lambda svl: svl.unit_cost if svl.quantity > 0 else -svl.unit_cost))"
```
Comment (carlosdauden) [inline stock_move_value_report/report/stock_move_line_value_report.xml:103]: Why don't use svl.value?

**Review by carlosdauden — CHANGES_REQUESTED** [review]

File: stock_move_value_report/report/stock_move_line_value_report.xml (hunk 1 above)
Comment (carlosdauden) [reply stock_move_value_report/report/stock_move_line_value_report.xml:103]: Take a look at this:
https://github.com/odoo/odoo/blob/1379970f916302aa7b5d2f5fb134f50baa62edec/addons/stock_account/models/product.py#L137-L158

Comment (CarlosRoca13) [reply stock_move_value_report/report/stock_move_line_value_report.xml:103]: this operation is for stock.move.lines not for stock.moves

Comment (carlosdauden) [reply stock_move_value_report/report/stock_move_line_value_report.xml:103]: Ok

**Review by carlosdauden — APPROVED** [review]

File: stock_move_value_report/report/stock_move_line_value_report.xml [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+                                    t-value="res_company.currency_id"
+                                />
+                            </t>
+                        </t>
+                    </t>
+                </div>
+                <!-- Total all moves -->
+                <div class="clearfix">
+                    <div id="total" class="row">
+                        <div class="col-4 float-right">
```
Comment (chienandalu) [inline stock_move_value_report/report/stock_move_line_value_report.xml]:
```suggestion
                        <div class="col-4 ml-auto">
```

File: stock_move_value_report/report/stock_move_line_value_report.xml [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+    </template>
+
+    <template id="stock_move_lines_value_total">
+        <div class="clearfix">
+            <div id="total" class="row">
+                <div class="col-4 float-right">
```
Comment (chienandalu) [inline stock_move_value_report/report/stock_move_line_value_report.xml]:
```suggestion
                <div class="col-4 ml-auto">
```

**Review by chienandalu — APPROVED** [review]
The total amount summary is aligned to the left. I tested this class that fixes it.

**Review by chienandalu — CHANGES_REQUESTED** [review]
Sorry, it was a request for changes, not an approval :)

**Comment by CarlosRoca13** [comment]
Changed @chienandalu

**Review by chienandalu — APPROVED** [review]
:+1:

**Pull request merged**